src/Scripting/Core/Hosting/ObjectFormatter/CommonPrimitiveFormatter.Options.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Scripting.Hosting
+{
+    public abstract partial class CommonPrimitiveFormatter
+    {
+        public struct Options
+        {
+            public readonly bool UseHexadecimalNumbers;
```

Comment:
Ah, I see this is already a part of PrintOptions.  I think that should be flowed through here.